tests/metrics/test_github.py

```diff
@@ -0,0 +1,169 @@
+from app.source.github import Github
+from app.daos.dao_git_metric import dao_upsert_git_metric
+from github3.exceptions import NotFoundError
+import datetime
+
+
+def mock_github_client(mocker, no_prs=False, create_date=datetime.datetime(2018, 12, 14)):
+    class MockGithubClient:
+        def __init__(self, token=''):
+            self.token = token
+
+        def search_repositories(self, query):
+            return [Repository(no_prs, create_date)]
```

Comment:
Don't need to pass in `no_prs` or `create_date` from the bottom, it can just be referred to in the function that needs it as it is should still be in scope.